src/patternfly/components/Content/content.scss

```diff
@@ -63,16 +63,15 @@
   --pf-c-content--a--hover--TextDecoration: var(--pf-global--link--TextDecoration--hover);
 
   // Blockquote
-  --pf-c-content--blockquote--Padding: var(--pf-global--spacer--md);
+  --pf-c-content--blockquote--PaddingX: var(--pf-global--spacer--md);
+  --pf-c-content--blockquote--PaddingY: var(--pf-global--spacer--md);
   --pf-c-content--blockquote--FontWeight: var(--pf-global--FontWeight--light);
   --pf-c-content--blockquote--Color: var(--pf-global--Color--200);
   --pf-c-content--blockquote--BorderLeftColor: var(--pf-global--BorderColor--light);
```

Comment:
I know I usually write `--pf-c-content--blockquote--BorderLeftColor` as 
`--pf-c-content--blockquote--BorderLeft--BorderColor` we should have a consistent approach , what are your thoughts?